src/scanpy/metrics/_gearys_c.py

```diff
@@ -182,7 +181,7 @@ def _gearys_c_vec_W(
 # https://github.com/numba/numba/issues/6774#issuecomment-788789663
 
 
-@numba.njit(cache=True)
+@numba.njit(cache=True, parallel=False)  # noqa: TID251
```

Comment:
This is an inner function. It wasn’t parallelized before and isn’t now. See also @ivirshup’s comment above